packages/services/f-braze-adapter/src/services/_tests/BrazeConsumer.test.js

```diff
@@ -71,18 +71,18 @@ describe('BrazeConsumer', () => {
         ${'customFilters'} | ${{ 'Consumer customFilters must be an Array': {} }}
     `('should throw an error if $parameter is $value', async ({ parameter, value }) => {
     // Arrange
-    function instantiateBrazeConsumer () {
-        const testOptions = {
-            ...mockConsumerOptions,
-            [parameter]: value[Object.keys(value)[0]]
-        };
-        // eslint-disable-next-line no-unused-vars
-        const consumer = new BrazeConsumer(testOptions);
-    }
+        function instantiateBrazeConsumer () {
```

Comment:
Minor thing but the comment above looks like it should also be aligned with the code?